### DISCUSSION TITLE
Upload an Artifact in Trial Version

### DISCUSSION THREAD

**Discussion by JustinDBruce** [keygen-sh/community#22] [General]
I'm working on a small sample project for proof of concept. I think I have most things figured out, but I not understanding how to upload and artifact in the dashboard. I'm looking under Distribution -> Artifacts and seeing no way to upload anything.  Thank you for the help.

**Comment by ezekg** [comment]
You can upload an artifact from a release's page by clicking the "Upload" button in the top right.

![image](https://user-images.githubusercontent.com/6979737/155422148-633faa9b-4720-4444-879e-04aff745c4b6.png)

Alternatively, you can use [the CLI](https://keygen.sh/docs/cli/).

**Reply by DRoberts532** [reply]
Hi Zeke,
Learning how to handle distributions.  The Upload button isn't appearing on my Releases page.  Did it move somewhere else, or get deprecated, or something else since the original question was posted?

**Reply by ezekg** [reply]
We revamped our distribution API earlier this year. Now, you create a release and then create and upload individual artifacts for the release. You can create a release from the Releases tab, and you can create artifacts from the Artifacts tab.

**Reply by DRoberts532** [reply]
I suspected it was something like that.  Thanks for the rapid response!  BTW, your product rocks.  We're creating licenses so easily for our software and I am so very glad I didn't have to do that from scratch.  Now its time for us to get our act together for update distributions.

**Reply by ezekg** [reply]
Great to hear! Let me know if you need anything else.
